### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by ShaguftaMethwani**
Minor grammatical errors

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ Now this service has been split into two parts:
 * `anitya <https://github.com/fedora-infra/anitya>`_: find and announce new
   releases
 * `the new hotness <https://github.com/fedora-infra/the-new-hotness/>`_:
-  listens to the fedmsg bus, open a ticket on bugzilla for packages allowing
+  listen to the fedmsg bus, open a ticket on bugzilla for packages allowing
```
Comment (pypingou) [inline README.rst]: the-new-hotness (it) listen*s*, that seems correct to me

**Comment by pypingou** [comment]
One remark otherwise looks good, thanks!

**Comment by ShaguftaMethwani** [comment]
Oh that's right !I have made it  listens. And similarly the other words in that sentence.

**Comment by ralphbean** [comment]
Looks good @ShaguftaMethwani.  Thanks!  :)

**Comment by pypingou** [comment]
Thanks @ShaguftaMethwani

**Comment by ShaguftaMethwani** [comment]
:)

**Comment by pypingou** [comment]
@ShaguftaMethwani I tried finding you in the Fedora account system to see if you had signed the FPCA but could not find you, could you please confirm that you gave us permission to use this change under the same license terms as the rest of the code in the project ?

Thanks

**Comment by pypingou** [comment]
@ShaguftaMethwani ping?

**Comment by pypingou** [comment]
@ShaguftaMethwani ping?
All I need is a reply saying: Yes, my contribution is licensed under the same license terms as the rest of the project.
It would be really nice if you could reply to this email with it :)

Thanks!

**Comment by pypingou** [comment]
Somehow github doesn't display @ShaguftaMethwani's reply so for the posterity here it is:
```
Date: Mon, 10 Aug 2015 17:39:22 +0530
From: shagufta methwani <methwanishagufta@gmail.com>
Subject: Reply to github ping for #122

  Yes  it is!

   I had mailed back preciously also when you had commented "ping?". Maybe
   some error. Anyway, emailing you directly. Sorry for the inconvenience.
```

Thanks @ShaguftaMethwani !  :)

**Pull request merged**